service/awsconfig/v4/resource/cloudformation/adapter/host_route_tables.go

```diff
@@ -82,7 +82,7 @@ func waitForPeeringConnectionID(cfg Config) (string, error) {
 			return microerror.Mask(err)
 		}
 		if len(output.VpcPeeringConnections) > 1 {
-			return microerror.Mask(tooManyResultsError)
+			return microerror.Maskf(tooManyResultsError, "too many peering connections")
```

Comment:
The error message will be like: `too many results: too many peering connections`. Looks like too many too manies :D